Source/Reactions/ReactorArkode.cpp

```diff
@@ -430,8 +430,8 @@ ReactorArkode::print_final_stats(void* arkode_mem)
   utils::check_flag(&flag, "ARKodeGetNumRhsEvals", 1);
 
 #ifdef AMREX_USE_OMP
-  amrex::Print() << "\nFinal Statistics: "
-                 << "(thread:" << omp_get_thread_num() << ", ";
+  amrex::Print() << "\nFinal Statistics: " << "(thread:" << omp_get_thread_num()
```

Comment:
avoid formatting change to otherwise unchanged file

Reply:
Removed the formatting. Thanks for pointing that out.